libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizer.cs

```diff
@@ -570,6 +570,38 @@ protected virtual async Task OnRecognizerResultAsync(RecognizerResult recognizer
             return Task.FromResult(properties);
         }
 
+        private static DelegatingHandler CreateHttpHandlerPipeline(HttpClientHandler httpClientHandler, params DelegatingHandler[] handlers)
```

Comment:
Why this has to be moved here?, isn't the `BuildLuisRecognizerOptionsV2` method also static?, unless there is something i cant think of that made you move it here

Reply:
You are right, I mixed up the methods a bit. Fixed. Thanks!